src/clients/java/src/test/java/com/tigerbeetle/BlockingRequestTest.java

```diff
@@ -483,22 +454,36 @@ public void testFailedCompletion() throws InterruptedException {
 
     }
 
+    private static NativeClient getDummyClient() {
+        var client = NativeClient.initEcho(0, "3000", 1);;
+
+        try {
+            // Calling close() to avoid the native client
+            // trying to release the dummy packet pointer.
+            client.close();
```

Comment:
I don't really understand this bit... If we must call close after `initEcho`, should we perhaps also do this in `public final class EchoClient implements AutoCloseable {` which I think is our public api?

Reply:
No, we don't!!! This code is obsolete, `client.close()` was to force the `if (contextHandle == 0L)` that [we discussed](https://github.com/tigerbeetledb/tigerbeetle/pull/905#discussion_r1241285744).